contrib/auto-completion/packages.el

```diff
@@ -202,7 +202,7 @@
 
       ;; add key into candidate list
       (setq helm-yas-display-key-on-candidate t)
-      (setq spacemacs--snippets-dir
+      (setq spacemacs--auto-completion-dir
             (concat (ht-get configuration-layer-paths 'auto-completion)
```

Comment:
Missed this on the previous commit, you can use `configuration-layers/get-layer-property 'auto-completion :dir` which should return the absolute path of the layer (auto-completion directory included, I believe).